cg_hermes/cli/validate.py

```diff
@@ -50,26 +50,26 @@ def validate_deliverables(
 
 
 @app.command("tags")
-def validate_tags_cmd(pipeline: Pipeline):
+def validate_tags_cmd(pipeline: Workflow):
     """Validate the tag maps for one of the definitions"""
     LOG.info("Validating %s common tags", pipeline)
     exit_code = 0
 
-    if pipeline == str(Pipeline.MIP_DNA):
+    if pipeline == str(Workflow.MIP_DNA):
```

Comment:
not sure if the casting to str is necessary considering that the class is a StrEnum

Reply:
Fixed